src/transaction/log_page_buffer.c

```diff
@@ -8108,18 +8072,15 @@ logpb_backup (THREAD_ENTRY * thread_p, int num_perm_vols, const char *allbackup_
   LOG_CS_ENTER (thread_p);
 
 #if defined(SERVER_MODE)
-
-  if (last_arv_needed < log_Gl.hdr.nxarv_num - 1)
+  if (first_arv_needed < log_Gl.hdr.nxarv_num - 1)
```

Comment:
log_Gl.hdr.nxarv_num(아직 생성되지 않은 다음 보관로그볼륨 ID) 보다 작은 경우엔 백업 볼륨에 포함되어야 합니다.
```suggestion
  if (first_arv_needed <= log_Gl.hdr.nxarv_num - 1)
```

Reply:
포함되도록 수정했습니다.